app/assets/stylesheets/components/indicators.scss

```diff
@@ -56,11 +56,7 @@
     var(--indicator-critical-color)
   );
 
-  // Circle, no text
```

Comment:
Apparently we don't really have bullet indicators (we had though in the past, for Connect).